diff --git a/build.sh b/build.sh
index 731d2a6..c25c54d 100755
--- a/build.sh
+++ b/build.sh
@@ -10,7 +10,7 @@ PLATFORMS=('unix' 'win32')
 BUILD_DIR=$PWD/_build
 SPECS="$SRC/config"
 ALIASES="$SRC/cursor-aliases"
-SIZES=('1' '1.25' '1.5' '2' '2.5' '3' '4' '5' '6' '10')
+SIZES=('1' '1.25' '1.5' '1.75' '2' '2.5' '3' '4' '5' '6' '10')
 DPIS=('lo' 'tv' 'hd' 'xhd' 'xxhd' 'xxxhd')
 SVG_DIM=24
 SVG_DPI=96
@@ -25,19 +25,19 @@ function set_sizes {
   max_size="$1"
   case $max_size in
     lo)
-      SIZES=("${SIZES[@]:0:3}")
+      SIZES=("${SIZES[@]:0:4}")
       ;;
     tv)
-      SIZES=("${SIZES[@]:0:4}")
+      SIZES=("${SIZES[@]:0:5}")
       ;;
     hd)
-      SIZES=("${SIZES[@]:0:5}")
+      SIZES=("${SIZES[@]:0:6}")
       ;;
     xhd)
-      SIZES=("${SIZES[@]:0:6}")
+      SIZES=("${SIZES[@]:0:7}")
       ;;
     xxhd)
-      SIZES=("${SIZES[@]:0:7}")
+      SIZES=("${SIZES[@]:0:8}")
       ;;
     xxxhd)
       SIZES=("${SIZES[@]}")
